scylla-gdb.py

```diff
@@ -3789,7 +3854,11 @@ def invoke(self, arg, for_tty):
                 for pclass_ptr in handles:
                     self._print_io_priority_class(fq_pclass(pclass_ptr), names_from_ptrs)
 
-            pending = circular_buffer(ioq['_sink']['_pending_io'])
+            try:
+                pending = chunked_fifo(ioq['_sink']['_pending_io'])
+                _ = len(pending)
```

Comment:
Why is this needed? Presumably that's the place that trows in case pending-io is not chunked fifo

Reply:
exactly. it's not necessary anymore. it was added for 
```py
try:
    pending = circular_buffer(ioq['_sink']['_pending_io'])
except gdb.error:
    # ..
```
as previously the constructor of `circular_buffer` does not throw. but since i put `chunked_info` in the `try` block, it does throw if the inspected variable's type does not match.

so, it's a leftover. will drop it.

Reply:
Here's how it is initialized now

```python
            try:
                pending = chunked_fifo(ioq['_sink']['_pending_io'])
            except gdb.error:
                pending = circular_buffer(ioq['_sink']['_pending_io'])
```

so it attempts to set pending as chunked fifo and falls back to circular buffer if it throws. But chunked fifo constructor is just

```python
    def __init__(self, ref):
        self._ref = ref
```

it doesn't do any type-checking, so why would it throw?

Reply:
sorry, i missed this. now i added a 
```py
_ = self.front_chunk
```
in the ctor. should be better now.